src/build2022.props

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why do we need this? do we have a special configuration?

Reply:
yes at the line 45/50, 'DEV17' symbol added. In build.props the symbol is DEV16. Thats the only difference between 2 files.